sldtk/detection.py

```diff
@@ -40,7 +40,7 @@ def detect_disk(img, threshold):
         raise TypeError("Expected single channel (grayscale) image.")
 
     blur = cv2.GaussianBlur(img, (5, 5), 0)
-    mask = cv2.inRange(blur, threshold, 255)
+    mask = cv2.inRange(blur, threshold, 2**16 - 1)
```

Comment:
Want to use np.iinfo(img.dtype).max in place of hardcoded values to conserve support for